rmgpy/rmg/react.py

```diff
@@ -73,12 +74,10 @@ def react(spc_fam_tuples, procnum=1):
 
     return reactions
 
-
```

Comment:
Could you undo the spacing changes? (two empty lines between top-level functions)

https://www.python.org/dev/peps/pep-0008/#blank-lines